src/api/java/appeng/api/config/FuzzyMode.java

```diff
@@ -24,9 +24,32 @@
 package appeng.api.config;
 
 public enum FuzzyMode {
-    // Note that percentage damaged, is the inverse of percentage durability.
-    IGNORE_ALL(-1), PERCENT_99(0), PERCENT_75(25), PERCENT_50(50), PERCENT_25(75);
+    /**
+     * Matches between 0% and 100% durability.
```

Comment:
`between` is not really defined as inclusive or exclusive and I can assume someone could consider it being exclusive here when mentioned explicitly like all other breakpoints.

Maybe just use `Matches any durability`?